src/components/dashboard/sidebar/SubsidiesSummary.jsx

```diff
@@ -132,6 +126,17 @@ const SubsidiesSummary = ({
     )
   );
 
+  const adminContactEmail = getContactEmail(enterpriseConfig);
+  const contactAdministratorCTA = (
+    <Button
```

Comment:
[suggestion/clarification] Should this button also rely on the same `variant={ctaButtonVariant}` used by `searchCoursesCta` above? The intent of `ctaButtonVariant` is supposed to de-emphasize this CTA if the learner has no course enrollments yet to not take attention away from the empty state messaging on the dashboard which includes at least one primary CTA that should have the user's primary attention.

Reply:
I think its a good idea to put it into the component now that a reasoning behind it was explained. Also it will maintain cohesion between the two CTA types (view a course, contact admin)